x/ibc/03-connection/handler.go

```diff
@@ -23,7 +23,6 @@ func HandleMsgConnectionOpenInit(ctx sdk.Context, k Keeper, msg MsgConnectionOpe
 		sdk.NewEvent(
 			sdk.EventTypeMessage,
 			sdk.NewAttribute(sdk.AttributeKeyModule, AttributeValueCategory),
-			sdk.NewAttribute(sdk.AttributeKeySender, msg.Signer.String()),
```

Comment:
Are these duplicates?

Reply:
Yes because they now will happen in antehandler. So no need to do it again here